app/assets/stylesheets/govuk_publishing_components/components/_input.scss

```diff
@@ -22,26 +22,19 @@ input[type=text].gem-c-input {
 
   // Disable inner shadow and remove rounded corners
   appearance: none;
+
```

Comment:
Moving these into main block after specificity was updated in https://github.com/alphagov/govuk_publishing_components/commit/19035c7df55db19d11373da00727e83801619aa6